packages/driver/cypress/fixtures/auth/delayedNavigate.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <button data-cy="to-localhost" onclick="setTimeout(()=>{window.location.href = 'http://localhost:3500/fixtures/auth/index.html'}, 1500)">navigate to localhost</button>
```

Comment:
It might be a bit easier to read the contents of these functions if we did something like a `script` tag that has a function that calls the setTimeout navigation